Tests/SmithyTimestampsTests/TimestampSerdeUtilsTests.swift

```diff
@@ -32,162 +32,64 @@ class TimestampSerdeUtilsTests: XCTestCase {
     // MARK: - Encoding Tests
 
     // Precision difference in linux documented in https://github.com/awslabs/aws-sdk-swift/issues/1006
```

Comment:
These tests still check timestamp logic, but don't do it through Codable logic.  As a result these tests are mush shorter & simpler.